scheduler-server/src/main/java/gov/nasa/jpl/aerie/scheduler/server/remotes/postgres/PostgresResultsCellRepository.java

```diff
@@ -0,0 +1,29 @@
+package gov.nasa.jpl.aerie.scheduler.server.remotes.postgres;
+
+import javax.sql.DataSource;
+
+import java.util.Optional;
+
+import gov.nasa.jpl.aerie.scheduler.server.ResultsProtocol;
+import gov.nasa.jpl.aerie.scheduler.server.models.SpecificationId;
+import gov.nasa.jpl.aerie.scheduler.server.remotes.ResultsCellRepository;
+
+public record PostgresResultsCellRepository(DataSource dataSource) implements ResultsCellRepository {
+  @Override
+  public ResultsProtocol.OwnerRole allocate(final SpecificationId planId)
```

Comment:
rename `planId` to `specificationId`  in this file.

Reply:
Done, thanks for catching this!